x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/import_rules_route.test.ts

```diff
@@ -0,0 +1,399 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { omit } from 'lodash/fp';
+
+import { getSimpleRuleAsMultipartContent } from '../../../../../../../../test/detection_engine_api_integration/security_and_spaces/tests/utils';
+import { ImportSuccessError } from '../utils';
+import {
+  getImportRulesRequest,
+  getImportRulesRequestOverwriteTrue,
+  getFindResult,
+  getResult,
+  getEmptyIndex,
+  getFindResultWithSingleHit,
+  getNonEmptyIndex,
+} from '../__mocks__/request_responses';
+import { createMockServer, createMockConfig, clientsServiceMock } from '../__mocks__';
+import { importRulesRoute } from './import_rules_route';
+
+describe('import_rules_route', () => {
+  let server = createMockServer();
+  let config = createMockConfig();
+  let getClients = clientsServiceMock.createGetScoped();
+  let clients = clientsServiceMock.createClients();
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+
+    server = createMockServer();
+    config = createMockConfig();
+    getClients = clientsServiceMock.createGetScoped();
+    clients = clientsServiceMock.createClients();
```

Comment:
Right here you will want this mock:

```ts
    clients = clientsServiceMock.createClients();
    clients.spacesClient.getSpaceId.mockReturnValue('default');
```

I almost argue that maybe the source ` clients.spacesClient` should be returning that default for _all_ mocks but up to you if you want to change it at the source or start just here since its weird that we have it returning `undefined` by default for when someone calls `getSpaceId` as that is usually the string `default`.

Reply:
👍 updated the mocks!